src/Ushahidi/Modules/V5/Commands/AbstractBaseCommandHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Commands;
+
+use App\Bus\Command\AbstractCommandHandler;
+use App\Bus\Command\command;
+use App\Bus\Action;
+use InvalidArgumentException;
+use Ushahidi\Modules\V5\Common\Errors;
+
+abstract class AbstractBaseCommandHandler extends AbstractCommandHandler
+{
+    use Errors;
+    
+    /**
+     * @param Command $command
+     * @return void
+     */
+    abstract protected function run(Command $command);
```

Comment:
I am not sure if it's a good idea. We should work with our interfaces not abstract classes to ensure a specific behaviour. I understand the reasoning but we have to think of a different way to introduce that idea.